invenio_accounts/admin.py

```diff
@@ -137,6 +140,56 @@ class RoleView(ModelView):
         list_all
 
 
+class SessionActivityView(ModelView):
+    """Admin view for user sessions."""
+
+    can_view_details = False
+    can_create = False
+    can_edit = False
+
+    list_all = ('user.id', 'user.email', 'sid_s', 'created')
+
+    column_labels = {
+        'user.id': 'User ID',
+        'user.email': 'Email',
+        'sid_s': 'Session ID',
+    }
+    column_list = list_all
+    column_filters = list_all
+    column_sortable_list = list_all
+
+    form_ajax_refs = {
+        'user': {
+            'fields': (User.id, User.email)
+        },
+    }
+
+    def delete_model(self, model):
+        is_current = SessionActivity.is_current(sid_s=model.sid_s)
+        if is_current:
```

Comment:
Just for the future reference, I would replace it with just:
`if SessionActivity.is_current(sid_s=model.sid_s)`
as you don't use this `is_current` variable anywhere else, so there is no benefit of assigning to this variable.
But that's just for the future, no need to change the PR because of this silly reason :)